myosuite/envs/myo/myochallenge/__init__.py

```diff
@@ -33,28 +33,53 @@ def register_env_with_variants(id, entry_point, max_episode_steps, kwargs):
 
 register_env_with_variants(id='myoChallengeBimanual-v0',
         entry_point='myosuite.envs.myo.myochallenge.bimanual_v0:BimanualEnvV1',
-        max_episode_steps=250,
+        max_episode_steps=300,
         kwargs={
             'model_path': curr_dir + '/../assets/arm/myoarm_bionic_bimanual.xml',
             'normalize_act': True,
             'frame_skip': 5,
+            'obj_scale_change': [0.1, 0.05, 0.1],  # 10%, 5%, 10% scale variations in respective geom directions
+            'obj_mass_change': (-0.050, 0.050),  # +-50gms
+            'obj_friction_change': (0.1, 0.001, 0.00002)  # nominal: 1.0, 0.005, 0.0001
         }
     )
 
+
 # MyoChallenge 2024 envs ==============================================
 register_env_with_variants(id='myoChallengeRunTrackP1-v0',
         entry_point='myosuite.envs.myo.myochallenge.run_track_v0:RunTrack',
         max_episode_steps=1000,
         kwargs={
             'model_path': curr_dir+'/../assets/leg/myoosl_runtrack.xml',
-            'normalize_act': True,
+            'normalize_act': False,
```

Comment:
normalize_act should be True. 
Is there is a good reason to change it to false? @P-Schumacher @elladyr

Reply:
The issue is that we have mixed actuators -> torque and muscle. @elladyr noticed that torque actuators would not be able to use their full force if this is turned on. So it was deactivated by default. 

Is there another strategy that's better?

Reply:
This was addressed in the branch [loco_actionspace_fix](https://github.com/MyoHub/myosuite/tree/loco_actionspace_fix)